### PR TITLE
Implementado diseño y provider para el listado de actividades

### PR DESCRIPTION
**Pull request by LordMario**
## Description
* Implementando diseño del listado de actividades
* Implementando provider para el listado de actividades

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (no functional changes, no api changes)
- [ ] This change requires a documentation update
- [ ] Other (please describe): 

## How Has This Been Tested?

Describe the tests you've run to verify the changes. Provide instructions for reproduction and indicate any relevant details for your test setup.

g
gg

New Change desing
ggg

New Change desing
aaaaa

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in areas hard to understand.
- [ ] I have made the corresponding changes to the documentation.
- [x] My changes do not generate new warnings.
- [x] I have added tests that prove my solution is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] All dependent changes have been merged and published in subsequent modules.
- [x] I have reviewed my code and corrected any spelling mistakes.

### CODE REVIEW

File: lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -72,10 +74,13 @@ class MessageLookup extends MessageLookupByLibrary {
             MessageLookupByLibrary.simpleMessage("Correo electrónico"),
         "Correo_o_nombre_de_usuario":
             MessageLookupByLibrary.simpleMessage("Correo o nombre de usuario"),
+        "Crear_nuevas_actividades":
+            MessageLookupByLibrary.simpleMessage("Crear nuevas actividades"),
```
Comment (SamBarFu) [inline lib/generated/intl/messages_en.dart:78]: Porqué esta en plural? acaso se van a crear varias actividades de un solo?

File: lib/generated/intl/messages_en.dart [hunk 2]
```diff
@@ -144,11 +156,14 @@ class MessageLookup extends MessageLookupByLibrary {
...
         "Ninos_asignados":
             MessageLookupByLibrary.simpleMessage("Niños asignados"),
+        "Nombre": MessageLookupByLibrary.simpleMessage("Nombre"),
         "Nombre_de_usuario":
             MessageLookupByLibrary.simpleMessage("Nombre de usuario"),
         "Observaciones": MessageLookupByLibrary.simpleMessage("Observaciones"),
         "Olvido_su_contrasena":
             MessageLookupByLibrary.simpleMessage("¿Olvido su contraseña?"),
+        "Opciones": MessageLookupByLibrary.simpleMessage("Opciones"),
+        "Pagina": MessageLookupByLibrary.simpleMessage("Pagina"),
```
Comment (SamBarFu) [inline lib/generated/intl/messages_en.dart:166]: Tildar palabra

File: lib/generated/intl/messages_es_NI.dart [hunk 3]
```diff
@@ -144,11 +156,14 @@ class MessageLookup extends MessageLookupByLibrary {
...
         "Ninos_asignados":
             MessageLookupByLibrary.simpleMessage("Niños asignados"),
+        "Nombre": MessageLookupByLibrary.simpleMessage("Nombre"),
         "Nombre_de_usuario":
             MessageLookupByLibrary.simpleMessage("Nombre de usuario"),
         "Observaciones": MessageLookupByLibrary.simpleMessage("Observaciones"),
         "Olvido_su_contrasena":
             MessageLookupByLibrary.simpleMessage("¿Olvido su contraseña?"),
+        "Opciones": MessageLookupByLibrary.simpleMessage("Opciones"),
+        "Pagina": MessageLookupByLibrary.simpleMessage("Pagina"),
```
Comment (SamBarFu) [inline lib/generated/intl/messages_es_NI.dart:166]: Tildar palabra

**Review by SamBarFu — CHANGES_REQUESTED** [review]

File: lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (LordMario) [reply lib/generated/intl/messages_en.dart]: Listo

File: lib/generated/intl/messages_en.dart (hunk 2 above)
Comment (LordMario) [reply lib/generated/intl/messages_en.dart]: listo

File: lib/generated/intl/messages_es_NI.dart (hunk 3 above)
Comment (LordMario) [reply lib/generated/intl/messages_es_NI.dart]: listo

**Review by SamBarFu — APPROVED** [review]

File: lib/presentation/pages/activities/activities.page.dart [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        getNextData: () {
+          listPatients.getNextActivities();
+        },
+        getPreviousData: () {
+          listPatients.getPreviusActivities();
+        },
+        headersRows: headersRowsActivity,
+        data: generateActivity(ref: ref),
+      ),
+      drawer: const SideMenu(),
```
Comment (SamBarFu) [inline lib/presentation/pages/activities/activities.page.dart:77]: Este SideMenu se esta mandando en todas las pantallas? porque no crear un layout?

**Review by SamBarFu — CHANGES_REQUESTED** [review]
Cambiar de posición el botón de agregar actividades, a una parte más visible y accesible para el usuario.

**Review by SamBarFu — APPROVED** [review]

**Pull request merged**